spinn_front_end_common/utility_models/live_packet_gather.py

```diff
@@ -44,10 +44,12 @@ def __init__(self, lpg_params, constraints=None):
 
     @overrides(ApplicationVertex.create_machine_vertex)
     def create_machine_vertex(
-            self, vertex_slice, resources_required,  # @UnusedVariable
+            self, vertex_slice, resources_required,
             label=None, constraints=None):
+        assert(vertex_slice is None)
+        assert(resources_required is None)
```

Comment:
@Christian-B Why did you put these `assert`s in? The general contract for `create_machine_vertex` allows these parameters to be other things (though in general we ought to know the slice and resources for the LPG prior to the call being made as LPGs are pretty simple).

This is causing a failure in `fec_integration_tests.interface.interface_functions.test_front_end_common_insert_edges_to_live_packet_gatherers.TestInsertLPGEdges`